polaris-react/rollup.config.js

```diff
@@ -0,0 +1,96 @@
+const {babel} = require('@rollup/plugin-babel');
```

Comment:
Would be nice to [add `JSDoc`s](https://github.com/Shopify/polaris/blob/66b43a6892cdac31d6884c4dfd431c4588705d75/polaris-tokens/rollup.config.js#L11-L13) in this file to get some type hints throughout the config. Can be done in a follow up task.